### ISSUE TITLE
add readme in tools/data whether the stata are for training sets only or for all data

### ISSUE THREAD

**Issue by jowagner** [UniversalDependencies/tools#82]
Users such as in shared tasks will need to know whether the stats and label sets in the `tools/data` folder are for just the training data or for all treebank data. A readme should be added.

**Comment by dan-zeman** [comment]
These are not statistics. These are internal data for the UD validator. README added.
